website/templates/emails/withdrawal_request_declined.html.mako

```diff
@@ -6,13 +6,28 @@
     <%!
         from website import settings
     %>
-        Dear ${requester.fullname},<br>
-        <br>
-        Your request to withdraw your ${document_type} <a href="${reviewable.absolute_url}">"${reviewable.title}"</a> from ${reviewable.provider.name} has been declined by the service moderators. Login and visit your ${document_type} to view their feedback. The ${document_type} is still publicly available on ${reviewable.provider.name}.
+        % if document_type == 'preprint':
+            Dear ${requester.fullname},<br>
+            <br>
+            Your request to withdraw your preprint <a href="${reviewable.absolute_url}">"${reviewable.title}"</a> from ${reviewable.provider.name} has been declined by the service moderators. Login and visit your ${document_type} to view their feedback. The ${document_type} is still publicly available on ${reviewable.provider.name}.
+            <br>
+            Sincerely,<br>
+            The ${reviewable.provider.name} and OSF Teams
+            <br>
+        % else:
+            Your request to withdraw your registration <a href="${reviewable.absolute_url}">"${reviewable.title}"</a> from ${reviewable.provider.name} has been declined by the service moderators. The registration is still publically available on ${reviewable.provider.name}.
+
+            % if comment:
```

Comment:
Is the template smart enough to interpret the empty string as false-y? Or do we need to use the `notify_comment` boolean from the context?